Uniswap/interface
# fix: show empty state on /add and /remove

Fixes behavior on /add and /remove to show an empty state when on unsupported chains.

## Screen capture
| Before       | After (Desktop)|
| ------------ |---------------- |
|<img width="1512" alt="Screen Shot 2023-04-19 at 1 01 27 PM" src="https://user-images.githubusercontent.com/4899429/233147890-f0a14f40-53b6-431e-a005-353dda5d84f0.png"> | <img width="1512" alt="Screen Shot 2023-04-19 at 1 02 11 PM" src="https://user-images.githubusercontent.com/4899429/233148018-53aa8c47-d707-4ff4-854a-74de37dc3cd5.png">|
|<img width="1198" alt="Screen Shot 2023-04-19 at 5 59 01 PM" src="https://user-images.githubusercontent.com/4899429/233209402-ac467574-e8e0-49b5-952b-97736da332b2.png">|<img width="1201" alt="Screen Shot 2023-04-19 at 5 59 49 PM" src="https://user-images.githubusercontent.com/4899429/233209415-2ada630c-1ff1-43e9-9c30-bd2bb0945479.png">|

## Test plan
### QA (ie manual testing)
1. Switch wallet to Sepolia
2. Go to app.uniswap.org/#/add
3. See the state in both the test preview and production

#### Devices
I tested this on mobile and desktop chrome.

### Automated testing
I do not think this is easy to test for the same reason as https://github.com/Uniswap/interface/pull/6338. Until we have cypress that can change chain ids, this is not easy to test automated. It's also a minimal change so manual testing here should be enough.
